packages/react-components/src/components/Date/SimpleDate.jsx

```diff
@@ -81,7 +91,9 @@ export const SimpleDate = ({
     <fieldset className={fieldsetClass} id={inputId.current}>
       <legend className="vads-u-font-size--base vads-u-font-weight--normal">
         {label || 'Date of birth'}
-        {required && <span className="form-required-span">(*Required)</span>}
+        {required && (
+          <span className="form-required-span">{i18next.t('required')}</span>
```

Comment:
This is the basic use of the `i18next` library - pass the `t()` function a key and it will return a value based on the language that is currently configured. Here's the [dependency documentation](https://www.i18next.com/overview/api#t).